build.py

```diff
@@ -134,6 +147,30 @@ def main(*argv):
             "--cache-dir", cache_dir
         ])
 
+        if hash_names:
+            for fn in os.listdir(out_tmp_dir):
+                fn = os.path.join(out_tmp_dir, fn)
+                if os.path.isfile(fn):
+                    print(
+                        "Computing hashes for \"%s\"." % os.path.basename(fn)
+                    )
+
+                    fn_hashes = {}
+                    for hn in hash_names:
+                        h = subprocess.check_output(
+                            ["openssl", hn, fn],
+                            universal_newlines=True
+                        )
```

Comment:
Tried reading the file and using `hashlib`, but that was way too slow (10mins+ as CI timed out). This was still the case even when no hashing was done. Shelling out to `openssl` seems to just be way more performant.